sig-instrumentation/charter.md

```diff
@@ -0,0 +1,69 @@
+# SIG Instrumentation Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Owns best practices for cluster observability through metrics and logging
+across all Kubernetes components and development of components required for all
```

Comment:
I'm still not sure "Kubernetes components" is a well understood, well defined term, but I will assume you are OK deferring the definition of this term to SIG Architecture and accepting whatever scope that implies if the definition is refined.

Reply:
i am fine w/ that caveat as described by @spiffxp